src/Settings.cpp

```diff
@@ -569,3 +577,46 @@ void Settings::restoreDefaults ()
     settings->clear();
     m_hCache.clear();
 }
+
+void Settings::exportSettings(const QString fileName)
+{
+    QSettings* exportSettings = new QSettings(fileName, QSettings::IniFormat);
+
+    const QStringList groups = settings->childGroups();
+    foreach(QString currentGroup, groups)
+    {
+        settings->beginGroup(currentGroup);
+        const QStringList keys = settings->childKeys();
+        foreach(QString currentKey, keys)
+        {
+            exportSettings->beginGroup(currentGroup);
+            exportSettings->setValue(currentKey, getValue((currentGroup.toStdString()), (currentKey.toStdString())));
+            exportSettings->endGroup();
+        }
+        settings->endGroup();
+    }
+}
+
+bool Settings::importSettings(const QString fileName)
+{
+    if(!verifyUserSettingsFile(fileName))
+        return false;
+
+    QSettings* importSettings = new QSettings(fileName, QSettings::IniFormat);
+
+    const QStringList groups = importSettings->childGroups();
+    foreach(QString currentGroup, groups)
```

Comment:
Do not use Qt's foreach, they can be considered obsolete after C++11 introduced [range-base for loops](https://en.cppreference.com/w/cpp/language/range-for).